src/Kernel/ClassDescription.class.st

```diff
@@ -33,12 +33,7 @@ ClassDescription >> addAndClassifySelector: selector withMethod: compiledMethod
 	priorMethodOrNil ifNotNil: [ priorOriginOrNil := priorMethodOrNil origin ].
 
 	oldProtocol := self organization protocolOfSelector: selector.
-	SystemAnnouncer uniqueInstance suspendAllWhile: [ "The next part is really ugly with the double if.. BUT! I will fix that in a new PR soon... I want to remove the usage of `Protocol unclassified` as a way to say 'Do not upatde the protocol' because this is counter intuitive."
```

Comment:
It just took a few weeks to clean the code enough to be able to remove those ugly ifs but I did it in the end :)